### PR TITLE
(IRO-2655) Added sync warning message.

### PR DESCRIPTION
**Pull request by azhurauski-sc**

### CODE REVIEW

**Review by skylarrichard123 — CHANGES_REQUESTED** [review]
I think it needs to stay until someone clicks the X button.

**Review by ndemidovich-sc — CHANGES_REQUESTED** [review]
Please add usage of Demo Data Manager.
You can use as example current implementation of [`NodeStatus` from `rpc` module at `ironfish`](https://github.com/iron-fish/ironfish/blob/d66edb4e6ff4b60a483c591eb450ddfd8ac8094b/ironfish/src/rpc/routes/node/getStatus.ts#L40) lib.

**Comment by skylarrichard123** [comment]
There's no preview link @ndemidovich-sc

**Comment by azhurauski-sc** [comment]
> There's no preview link @ndemidovich-sc

Fixed

File: src/providers/DataSyncProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      setSyncInterval(infinite)
+    } else if (
+      status?.blockSyncer.status === 'idle' ||
+      status?.blockSyncer.status === 'stopped'
+    ) {
+      setLoaded(true)
+      syncInterval && clearInterval(syncInterval)
+    }
+    return () => syncInterval && clearInterval(syncInterval)
+  }, [status?.blockSyncer.status])
```
Comment (ndemidovich-sc) [inline src/providers/DataSyncProvider.tsx:57]: I don't think that it is right way. For example, we synchronized node, but in 10 minutes it's begin sync again.

**Review by ndemidovich-sc — CHANGES_REQUESTED** [review]

**Review by skylarrichard123 — APPROVED** [review]

**Review by ndemidovich-sc — APPROVED** [review]

**Pull request merged**